src/platform/datapath_raw_xdp.c

```diff
@@ -1138,6 +1144,26 @@ CxPlatDpRawUninitialize(
     }
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+VOID
+CxPlatDpRawSetPortBit(
+    _Inout_ uint8_t *BitMap,
+    _In_ uint16_t Port
+    )
+{
+    BitMap[Port >> 3] |= (1 << (Port & 0x7));
```

Comment:
Should these be interlocked functions?

Reply:
We are doing it under rule lock. We need to hold the lock for iterating all rules to find the right rule for the connection anyways.

Reply:
Ok. Let's go with it. If we need anything else later, we can tackle it then.